doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multi-Orchestrator Example
+==========================
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands (necessitating a co-located deployment),
+along with an analysis and
+visualization workflow connected to the simulation (requiring a standard orchestrator).
+
+Below is a simple example demonstrating the process of:
+
+1. Launch two standard Orchestrators within the experiment with unique identifiers
+2. Launch one colocated Orchestrator within the experiment with a unique identifier
+3. Connect SmartRedis clients within the driver script to send tensors to std Orchestrators
+4. Connect SmartRedis clients within the application to retrieve tensors from std databases to store in colo databases
+
+The Application Script
+----------------------
+To store and retrieve data to and from databases
+during the course of an application, you must
+initialize multiple SmartSim clients (SmartRedis).
+In this section, we write an application script
+to demonstrate how to connect SmartRedis
+clients in the context of multiple
+launched databases. Using the clients, we retrieve tensors
+from two databases launched in the driver script, then store
+the tensors in the colocated database.
+
+To begin, import the necessary packages:
+
+.. code-block:: python
+
+  from smartredis import ConfigOptions, Client, log_data
```

Comment:
It looks like you're using Client.log_data(), and not the standalone log_data() function

Reply:
hmmm could you elaborate? I do want to use the Client.log_data() function here?

Reply:
Unless I missed it, you're not using the standalone log_data() method, so don't need to import it